doc/api/deprecations.md

```diff
@@ -511,6 +511,14 @@ Type: Documentation-only
 The `fs.SyncWriteStream` class was never intended to be a publicly accessible
 API.
 
+<a id="DEP0062"></a>
+### DEP0062: node --debug
+
+Type: Runtime
+
+`--debug` activates the legacy V8 debugger which is to be removed in favor of
+Inspector.
```

Comment:
This can be expanded a bit to point out the use of the `--inspect` alternative and the reasons why `--debug` is being removed.